testing/scripts/test_api.sh

```
@@ -1027,6 +1027,86 @@ else
+    assert_http 404 "GET deleted device returns 404"
+else
+    skip "Device sub-tests — no DEV_ID"
+fi
+
+# ====================================================================
+log "=== OpenAPI Spec ==="
+# ====================================================================
+
+# GET spec (no auth required)
+SPEC_HTTP=$(curl -s --noproxy '*' -k -o /tmp/_ipam_spec.yaml -w '%{http_code}' "$BASE_URL/api.php?resource=spec")
+[[ "$SPEC_HTTP" == "200" ]] && pass "GET ?resource=spec HTTP 200" || fail "GET ?resource=spec HTTP $SPEC_HTTP"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)